libvsi.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=/usr/local
+exec_prefix=${prefix}
+includedir=${prefix}/include
+libdir=${exec_prefix}/lib
+
+Name: libvsi
+Description: Distributes vehicle signals between components inside the IVI
```

Comment:
Please change to something more generic.  This library does not need to be limited to "IVI".

Perhaps something like:

Access library for distributing vehicle signals from low-level signal providers to client software components